android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/SessionState.kt

```diff
@@ -27,6 +28,7 @@ interface SessionState {
     val id: String
     val content: ContentState
     val trackingProtection: TrackingProtectionState
+    val cookieBannerStatus: CookieBannerHandlingStatus
```

Comment:
@iorgamgabriel we are going to expose this new state, that you will be able to observe!

Reply:
it will work similar to the `TrackingProtectionState`